package.json

```diff
@@ -43,23 +43,23 @@
     "@babel/preset-env": "^7.3.1",
     "@babel/preset-react": "^7.0.0",
     "@octokit/rest": "^16.28.6",
-    "@paprika/button": "^0.1.0",
-    "@paprika/date-picker": "^0.4.0",
-    "@paprika/heading": "^0.1.0",
+    "@paprika/button": "^0.2.0",
```

Comment:
this sounds fishy to me. Sounds like relying on package hoisting.
I think the ideal solution would be when we extract stories into a separate package, that package could depend on everything else

Reply:
Fishy indeed. 🤨 🐟 
By "when we extract stories into a separate package" do you mean Danil's suggestion to have a separate Storybook package?

Reply:
yeah, it sounded like people were in favour of that?

Reply:
I don't think it is fish, Lerna goes into each package and checks for the versions, compares them and if two versions need to exists leave a node_modules on the package and hoist the other up to the root level.

If we set a specific package then we have a problem when you update two packages at the same time because the resolution of the version for storybook will be the one declare at the top of the package, making harder to the developer to figure out what is happening.

As an example, if you modify the L10n component you can't see any change unless you manually bump the version on the storybook, *`npx lerna bootstrap` will stop working*. 

And for the CI, we are preserving the node_modules across the testings so once we run yarn and lerna do their job hoisting their package we are set.

I would say that this is actually the behaviour that we want and not lock the packages at root level.